pkg/webui/console/containers/devices-table/index.js

```diff
@@ -57,6 +61,11 @@ import {
 
 import style from './devices-table.styl'
 
+const m = defineMessages({
+  otherClusterTooltip:
+    'This end device is registered on a different cluster or host. It cannot be accessed using this Console. To access this device, use the Console of the cluster that this device was registered on.',
```

Comment:
@kschiffer  Does it makes sense to tell the user where the end device is registered?

Reply:
Yeah, lets add it like this

```suggestion
    'This end device is registered on a different cluster or host (`{host}`). It cannot be accessed using this Console. To access this device, use the Console of the cluster that this device was registered on.',
```

Reply:
It makes sense indeed. 
I have a question related this. Some devices don't provide all servers. For example, a GET request to this end device: `/api/v3/applications/adrian-test-lab/devices/eui-0004a310001ab0b6` with the field masks: `name,description,version_ids,network_server_address,application_server_address,join_server_address,locations,attributes` responds with an object where "ns" and "as" are the same as host but that doesn't contain a `join_server_address`. 

That's why according to the logic implemented on device general settings the join server field appears blocked with the message "Not registered in this cluster" as you can see here:
<img width="693" alt="Screenshot 2021-12-10 at 12 44 08" src="https://user-images.githubusercontent.com/56658938/145576053-b27d7140-fd96-42c4-8b42-19f4ece3773b.png">

Does it make sense to block the device in the device table, implementing the same logic? For me a check for if the ns, as and js are defined before comparing them to the current host would make more sense. Because the information of join server address not being provided could mean that the device is simply not registered on that server and not that it's on another cluster. Or am I getting something wrong?

Reply:
This is a Generic Node that is provisioned on an external Join Server. Such end devices should not be hidden because they are still originating from the current cluster or host (unless both AS and NS are registered on different clusters).

See @htdvisser's remark here on when to determine end devices to be on "Other cluster": [https://github.com/TheThingsIndustries/lorawan-stack-support/issues/385#issuecomment-879677908](https://github.com/TheThingsIndustries/lorawan-stack-support/issues/385#issuecomment-879677908)

> For end devices this would happen if all of \`network\_server\_address\`, \`application\_server\_address\` and \`join\_server\_address\` are a different cluster than current

Note the logical AND here. Any server address not being present should remove them from the check.

E.g., an end device that does not have `join_server_address` set should only check for `network_server_address` and `application_server_address` to be different.